lab_2_keywords_cooccurrence/main.py

```diff
@@ -161,4 +225,44 @@ def load_stop_words(path: Path) -> Optional[Mapping[str, Sequence[str]]]:
     :param path: path to the file with stop word lists
     :return: a dictionary containing the language names and corresponding stop word lists
     """
-    pass
+    if not type_check(path, Path):
+        return None
+    with open(path, 'r', encoding='utf-8') as file:
+        return dict(load(file))
```

Comment:
better - `json.load`. Why do you cast to dict?

Reply:
This IS `json.load`. Line 8 of the same file: `from json import load`. Casting to dict removed one of the check types mistakes. I will try to find a way to remove it if possible though.

Reply:
Yes, but `yaml` has the same function. And I can write my `load` function. But if you import only `json` and will use `json.load` it will be more clear, that this load is the load function  from json

Reply:
Fixed